evap/static/ts/src/datagrid.ts

```diff
@@ -141,10 +143,16 @@ abstract class DataGrid {
             });
             const isDisplayedByFilters = [...this.state.filter].every(([name, filterValues]) => {
```

Comment:
It would probably make sense to explicitly name these ("equality filters" and "range filters"), otherwise I can see this becoming confusing.